src/main/resources/application.yml

```diff
@@ -35,8 +35,13 @@ spring:
           starttls:
             enable: true
             required: true
-          from: no-reply@linchpino.com
-mail.from.name: Linchpino
+
+email:
+  from:
+    name: Linchpino
+    mailAddress: no-reply@linchpino.com
+  jobSeekerSubject: Confirmation of Interview Schedule on Linchpino
+  mentorSubject: Welcome to Linchpino - Confirmation of Mentor Registration
```

Comment:
We can hardcode these properties. In future maybe we need to have a localization phase